src/modules/person/api/person.frontend.controller.spec.ts

```diff
@@ -102,5 +114,57 @@ describe('PersonFrontendController', () => {
             expect(result.items.length).toEqual(2);
             expect(result.items.at(0)?.person.name.vorname).toEqual('Max');
         });
+
+        it('should get a person with the given rolle id', async () => {
+            const personPermissions: DeepMocked<PersonPermissions> = createMock();
+            personPermissions.getOrgIdsWithSystemrecht.mockResolvedValueOnce([personController.ROOT_ORGANISATION_ID]);
+
+            personRepositoryMock.findBy.mockResolvedValue([[person1], 1]);
+            const rolleID: string = personenkontext1.rolleId;
+            const result: PagedResponse<PersonendatensatzResponse> = await personController.findPersons(
+                { ...queryParams, rolleID },
+                personPermissions,
+            );
+
+            expect(personRepositoryMock.findBy).toHaveBeenCalledTimes(1);
+            expect(result.total).toEqual(1);
+            expect(result.limit).toEqual(1);
+            expect(result.offset).toEqual(0);
+            expect(result.items.length).toEqual(1);
+            expect(result.items.at(0)?.person.name.vorname).toEqual('Max');
+            expect(result.items.at(0)?.person.id).toEqual(person1.id);
+        });
+
+        it('should get a person with the given orgnisation id', async () => {
```

Comment:
Müsste dieser Test nicht fehlschlagen? Der Mock gibt ein Array (organisationIDs) zurück was nur die RootID enthält, aber du fragst eine ander ID ( personenkontext1.organisationId) ab. Demenstprechend müsste doch  organisationIDs.includes(queryParams.organisationID) false sein und damit ein Not Authorized geworfen werden...?